projects/ngx-sub-form/src/lib/ngx-sub-form-utils.ts

```diff
@@ -32,6 +34,37 @@ export type FormErrors<FormInterface> = null | Partial<
   }
 >;
 
+type Omit<T, K> = Pick<T, Exclude<keyof T, K>>; // can be removed when upgrading ts version
+type UntypedControlBase<TControl extends AbstractControl | FormArray> = Omit<
+  TControl,
+  'value' | 'valueChanges' | 'setValue' | 'patchValue'
+>;
+
+export type TypedAbstractControl<TValue extends any> = {
+  value: TValue;
+  valueChanges: Observable<TValue>;
+  setValue(
+    value: TValue,
+    options?: {
+      onlySelf?: boolean;
+      emitEvent?: boolean;
+      emitModelToViewChange?: boolean;
+      emitViewToModelChange?: boolean;
+    },
+  ): void;
+  patchValue(
+    value: Partial<TValue>,
+    options?: {
+      onlySelf?: boolean;
+      emitEvent?: boolean;
+      emitModelToViewChange?: boolean;
+      emitViewToModelChange?: boolean;
```

Comment:
Same as setValue. We could just have `Parameters<FormControl['patchValue']>[1]`

Reply:
done, but see comments [below](https://github.com/cloudnc/ngx-sub-form/pull/139#issuecomment-586591629)